### PR TITLE
OK

### PR DESCRIPTION
**Pull request by Krzysiek31415**

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: schedule
🏅 Granted 12 XP for @Krzysiek31415

File: homework/schedule/schedule.hpp [hunk 1]
```diff
@@ -0,0 +1,29 @@
+#pragma once 
+
+
+#include <chrono>
+#include <functional>
+#include <thread>
+#include <iostream>
+#include <string>
```
Comment (lukasz-kukulka) [inline homework/schedule/schedule.hpp:8]: Alphabetic

File: homework/schedule/schedule.hpp [hunk 2]
```diff
@@ -0,0 +1,29 @@
+#pragma once 
+
```
Comment (lukasz-kukulka) [inline homework/schedule/schedule.hpp:2]: Redundant line

**Comment by ziobron** [comment]
Thanks @lukasz-kukulka for Code Review.
🏅 1 XP granted

**Pull request closed without merging**